game/view/actor.lua

```diff
@@ -28,7 +28,7 @@ local function _initGraphicValues()
   _exptext = "EXP: %d"
   _statstext = "STATS\nATH: %d\nARC: %d\nMEC: %d\nSPD: %d"
   _depthtext = "DEPTH: %d"
-  _buffertext = "BUFFER #%d: %d cards [%d remembered]"
+  _buffertext = "DECK: %d cards [%d in discard pile]"
```

Comment:
The in-game term for "deck" is still "buffer", right?

Reply:
Oh, right. I should fix that. I might have done this without thinking